### PR TITLE
Bliss

### PR DESCRIPTION
**Pull request by isuruf**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by ocefpaf** [comment]
Minor comments that can be addressed/answered later. Merging this!

**Comment by isuruf** [comment]
@ocefpaf, did you forget to submit the comments?

**Comment by ocefpaf** [comment]
> @ocefpaf, did you forget to submit the comments?

Yep. Bad connection :-/
Looks like they are up now.

File: recipes/bliss/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+libtoolize
+autoreconf -fi
+automake --add-missing --copy
+./configure --prefix=$PREFIX --disable-gmp
```
[the remark this answers is not in the document]
Comment (isuruf) [reply recipes/bliss/build.sh]: No. Results in segfaults when gmp is enabled

**Pull request merged**